src/screens/WhotoFollow/elements/ItemUser.js

```diff
@@ -31,11 +31,7 @@ const ItemUser = ({photo, username, bio, followed, onPress, userid, karmaScore})
       <View style={styles.containerButton}>
         <Pressable onPress={onPress} style={styles.followAction(24, 24)}>
           <Animated.View style={{position: 'absolute', opacity: 1}}>
-            <IconAdd
-              width={dimen.normalizeDimen(24)}
-              height={dimen.normalizeDimen(24)}
-              fill="#000000"
-            />
+            <IconAdd width={dimen.normalizeDimen(24)} height={dimen.normalizeDimen(24)} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `fill` prop for the `IconAdd` component has been removed in the new hunk. If this was intentional and the default fill color of the `IconAdd` component is being used, then it's fine. However, if you want to maintain the previous behavior where the fill color was explicitly set to `#000000`, you should add it back.
```diff
- <IconAdd width={dimen.normalizeDimen(24)} height={dimen.normalizeDimen(24)} />
+ <IconAdd width={dimen.normalizeDimen(24)} height={dimen.normalizeDimen(24)} fill="#000000" />
```
Please ensure that this change does not affect the visual appearance of the icon in a way that is unintended.


<!-- This is an auto-generated comment by OSS CodeRabbit -->